geode-serialization/src/main/java/org/apache/geode/internal/serialization/filter/ReflectiveFacadeObjectInputFilter.java

```diff
@@ -78,4 +77,36 @@ public String toString() {
   ObjectInputFilterApi getObjectInputFilterApi() {
     return api;
   }
+
+  private void handleExceptionThrownByApi(ReflectiveOperationException e)
```

Comment:
this looks like repeated code with `ReflectiveFacadeGlobalSerialFilter`, anyway to consolidate them?

Reply:
I can combine them into a new ErrorHandler class.

Do the changes to ManagementAgent look ok?

Reply:
Yes